src/main/jjtree/bsh.jjt

```diff
@@ -1283,18 +1327,6 @@ void BlockStatement() :
 |
   // Allow BeanShell package declarations in any block
   PackageDeclaration()
-|
-  FormalComment()
-}
-
-void FormalComment() #FormalComment( retainComments ) :
```

Comment:
Ok so we are turning FormalComment into a Node and not a Token anymore.

Not sure I follow the reasoning here, please explain.

Reply:
My bad